helios-services/src/main/java/com/spotify/helios/servicescommon/statistics/MasterMetricsImpl.java

```diff
@@ -75,6 +77,19 @@ private MasterRequestMetrics request(final String name) {
   }
 
   @Override
+  public void clientVersion(final String version) {
+    final String name = name(group, TYPE + "_meter_client_version", version);
+
+    Meter m = clientVersionMeters.get(name);
+    if (m == null) {
+      m = registry.meter(name);
+      clientVersionMeters.put(name, m);
```

Comment:
since the MetricRegistry maintains a ConcurrentHashMap for all of the registered metrics, and examines it when you call `register.meter(name)`, feels like you could do away with the same map-keeping here, as it is pretty unlikely to try to request a metric with this name that isn't a meter